invoice_department/__init__.py

```diff
@@ -1 +1,3 @@
+# -*- coding: utf-8 -*-
 from . import invoice
```

Comment:
Can you create a new directory for models and update the code to the new api in the function...